### PR TITLE
Fix signal kyco/core.shutdown

### LINKED ISSUE #101: Check controller stop action
Today controller.stop() should send a signall to all napps and shutdown his own thread. Maybe this is not working properly.

### PR DESCRIPTION
**Pull request by macartur**
- Fix signal shutdown in core.napps.py
- Add method _execute_loop to run execute method until the signal
  'kyco/core.shutdown' is received.
- Remove unused comments.
- Fix #101

### CODE REVIEW

**Comment by macartur** [comment]
After send a shutdown some times the thread can't stop. I will investigate it.

**Comment by macartur** [comment]
@raphaelmcobe , I discovered the error, the package kyco-core-napps has a older implementation using signals with Classes and now we have a specific string to represent a signal. If you running locally all tests pass.

File: kyco/core/napps.py [hunk 1]
```diff
@@ -48,12 +50,18 @@ def run(self):
...
+        self._execute_loop()
 
-    @listen_to('KycoShutdownEvent')
+    @listen_to('kyco/core.shutdown')
     def _shutdown_handler(self, event):
+        self.__event.set()
         self.shutdown()
 
+    def _execute_loop(self):
+        while(not self.__event.is_set()):
```
Comment (cemsbr) [inline kyco/core/napps.py]: What if the user only want to run `execute()` once?

**Review by cemsbr — CHANGES_REQUESTED** [review]
I'm taking care of the requested changes.

**Review by cemsbr — APPROVED** [review]

**Pull request merged**